client/js/components/Navigation.jsx

```diff
@@ -3,12 +3,14 @@ import React from 'react';
 
 export default class Navigation extends React.Component {
     render () {
+        var h1 = <h1 onClick={() => window.location.reload()}>Austin Green Map</h1>;
```

Comment:
can this be named something more descriptive than h1? maybe `siteTitle` or  `home` or `defaultNoChildren`

also, trying to learn, is this a place where `const` could be used?